soc/nxp/imxrt/imxrt7xx/Kconfig.defconfig

```diff
@@ -0,0 +1,47 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_MIMXRT798S_HIFI4
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 325000000
+
+config SMP
+	default n
+
+config XTENSA_TIMER
+	default y
+
+config MULTI_LEVEL_INTERRUPTS
+	default n
+
+config 2ND_LEVEL_INTERRUPTS
+	default n
+
+# To prevent test uses TEST_LOGGING_MINIMAL
+config TEST_LOGGING_DEFAULTS
+	default n
+	depends on TEST
+
+config XTENSA_CCOUNT_HZ
+	default SYS_CLOCK_HW_CYCLES_PER_SEC
+
+config SYS_CLOCK_TICKS_PER_SEC
+	default 1000
+
+config DYNAMIC_INTERRUPTS
+	default n
+
+config CACHE
+	default n
+
+config DCACHE
+	default n
+
+config CACHE_MANAGEMENT
+	default n
```

Comment:
These are disabled by default so there is no need to set default to 'n'. Ditto for `SMP` and `*_LEVEL_INTERRUPTS` above.